src/thunderbots/software/ai/hl/stp/evaluation/pass.h

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Can this file be changed to something like `intercept` for clarity? This isn't passing-specific.

Reply:
renamed and changed file headers